docs/controls/Loading.md

```diff
@@ -0,0 +1,60 @@
+# Loading XAML Control 
+
+The loading control is for showing an animation with some content when the user should wait in some tasks of the app.
+
+## Syntax
+
+An example of how we can build the loading control.
+
+``` xaml
+<controls:Loading x:Name="LoadingControl" LoadingHorizontalAlignment="{Binding LoadingContentHorizontalAlignment.Value}" 
+                    LoadingVerticalAlignment="{Binding LoadingContentVerticalAlignment.Value}"
+                    LoadingBackground="{Binding LoadingBackground.Value}" LoadingOpacity="{Binding OpacityForLoadingControl.Value}">
+    <ContentControl x:Name="LoadingContentControl"/>
+</controls:Loading>
+```
+- **LoadingHorizontalAlignment** and **LoadingVerticalAlignment** are responsible to align the custom content of the control. It aligns what we create inside as a ContentPresenter.
```

Comment:
Docs need to be update with regard to the Loading* properties: eg: LoadingHorizontalAlignment